static/js/components/LearnerSearch.js

```diff
@@ -128,6 +129,16 @@ export default class LearnerSearch extends SearchkitComponent {
       {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
       LearnerResult
     );
+    this.forceSearch = true;
+  }
+
+  loaded = (): boolean => {
+    if (this.forceSearch) {
```

Comment:
I'm not sure I exactly understand what's going on here - it looks like we'll only have `this.forceSearch === true` on the first render, after which we'll set it to false (on the line below).

Why is it that we can't just call `this.isInitialLoading`, and why, in the case where we're doing our first render, we would want to return `true` here automatically (on line 138)?